src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,26 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DeleteHistories {
+
+    private final List<DeleteHistory> deleteHistories;
+
+    public DeleteHistories(DeleteHistory... deleteHistories) {
+        this.deleteHistories = new ArrayList<>(Arrays.asList(deleteHistories));
```

Comment:
```suggestion
        this(Arrays.asList(deleteHistories));
```
생성자 한 곳에서 관리하도록 만들면 어떨까요

Reply:
아.. 이부분도 항상 고민하던 부분이었는데요. 
생성자를 여러개 만들경우 가능하면 한곳에서 처리가능하도록 하는것이 좋은것이었군요.